### PR TITLE
set chat tab as default for game channel

### PR DESCRIPTION
**Pull request by jonager**

### CODE REVIEW

File: liwords-ui/src/chat/chat.tsx [hunk 1]
```diff
@@ -329,10 +329,10 @@ export const Chat = React.memo((props: Props) => {
   }, [defaultChannel, defaultDescription, loggedIn, props.suppressDefault]);
 
   useEffect(() => {
-    if (props.tournamentID) {
+    if (props.tournamentID || channelType === 'game') {
```
Comment (domino14) [inline liwords-ui/src/chat/chat.tsx:332]: let's add `gametv` as a possible channel type for this too

**Review by domino14 — CHANGES_REQUESTED** [review]
let's do it for gametv too (for observers of ongoing games)

File: liwords-ui/src/chat/chat.tsx (hunk 1 above)
Comment (jonager) [reply liwords-ui/src/chat/chat.tsx]: done, maybe it's a good idea to create a channelType enum if we start using it in other places

**Review by domino14 — APPROVED** [review]

**Comment by domino14** [comment]
merged copy branch #1076

**Pull request closed without merging**